### PR TITLE
Rename "SentinelOne Event Collector" to "SentinelOne Activity and Alerts"

### PR DESCRIPTION
**Pull request by mmhw**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
Related: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-11847)

## Description
Rename "SentinelOne Event Collector" to "SentinelOne Activity and Alerts".

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/SentinelOne/Integrations/SentinelOneEventCollector/SentinelOneEventCollector.yml [hunk 1]
```diff
@@ -54,7 +54,7 @@ configuration:
   section: Collect
   required: false
 description: This integration fetches activities, threats, and alerts from SentinelOne.
-display: SentinelOne Event Collector
+display: SentinelOne Activity and Alert Collector
```
Comment (mmhw) [inline Packs/SentinelOne/Integrations/SentinelOneEventCollector/SentinelOneEventCollector.yml:57]:
```suggestion
display: SentinelOne Activity and Alerts Collector
```

**Review by nkanon — CHANGES_REQUESTED** [review]
Please remove "collector" from the name.
It should be: "SentinelOne Activity and Alerts"

Thanks!

File: Packs/SentinelOne/Integrations/SentinelOneEventCollector/README.md [hunk 2]
```diff
@@ -3,10 +3,10 @@ This integration was integrated and tested with API version 2.1 of SentinelOne.
 
 This is the default integration for this content pack when configured by the Data Onboarder in Cortex XSIAM.
 
-## Configure SentinelOne Event Collector on Cortex XSIAM
+## Configure SentinelOne Activity and Alerts Collector on Cortex XSIAM
```
Comment (mmhw) [inline Packs/SentinelOne/Integrations/SentinelOneEventCollector/README.md:6]:
```suggestion
## Configure SentinelOne Activity and Alerts on Cortex XSIAM
```

File: Packs/SentinelOne/Integrations/SentinelOneEventCollector/README.md [hunk 3]
```diff
@@ -3,10 +3,10 @@ This integration was integrated and tested with API version 2.1 of SentinelOne.
...
 
 1. Navigate to **Settings** > **Configurations** > **Data Collection** > **Automation & Feed Integrations**.
-2. Search for SentinelOne Event Collector.
+2. Search for SentinelOne Activity and Alerts Collector.
```
Comment (mmhw) [inline Packs/SentinelOne/Integrations/SentinelOneEventCollector/README.md:9]:
```suggestion
2. Search for SentinelOne Activity and Alerts.
```

File: Packs/SentinelOne/Integrations/SentinelOneEventCollector/SentinelOneEventCollector.yml [hunk 4]
```diff
@@ -54,7 +54,7 @@ configuration:
   section: Collect
   required: false
 description: This integration fetches activities, threats, and alerts from SentinelOne.
-display: SentinelOne Event Collector
+display: SentinelOne Activity and Alerts Collector
```
Comment (mmhw) [inline Packs/SentinelOne/Integrations/SentinelOneEventCollector/SentinelOneEventCollector.yml:57]:
```suggestion
display: SentinelOne Activity and Alerts
```

File: Packs/SentinelOne/ReleaseNotes/3_2_32.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### SentinelOne Activity and Alerts Collector
```
Comment (mmhw) [inline Packs/SentinelOne/ReleaseNotes/3_2_32.md:4]:
```suggestion
##### SentinelOne Activity and Alerts
```

File: Packs/SentinelOne/ReleaseNotes/3_2_32.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
...
+
+- Renamed the integration **SentinelOne Event Collector** to **SentinelOne Activity and Alerts Collector**.
```
Comment (mmhw) [inline Packs/SentinelOne/ReleaseNotes/3_2_32.md:6]:
```suggestion
- Renamed the integration **SentinelOne Event Collector** to **SentinelOne Activity and Alerts**.
```

**Review by nkanon — APPROVED** [review]

**Pull request merged**